tests/rubinrepoman_test.py

```diff
@@ -4,17 +4,66 @@
 registry, getting the list of available tags, and filtering
 that down into a list of images to pull.
 """
-
 from __future__ import annotations
 
 from typing import List
 
 from cachemachine.rubinrepoman import RubinRepoMan
-from cachemachine.types import CachedDockerImage
+from cachemachine.types import CachedDockerImage, DockerImage
 
+from .conftest import mock_registry
 from .docker_mock import DockerMock
 
-HOST = "registry.hub.docker.com"
+repo = "registry.hub.docker.com/lsstsqre/sciplat-lab"
+expected_names = {
+    "prepuller_pulled_recommended": "prepuller_pulled_recommended",
```

Comment:
Because RubinTagType.UNKNOWN tags don't even get titlecased.  In some sense it doesn't matter anyway since the only place it will ever show up (since UNKNOWN) is in the dropdown, where it will get the tag name rather than the display name anyway.

Reply:
Ah, yes, makes sense.